samples/highcharts/annotations/defer/demo.js

```diff
@@ -0,0 +1,41 @@
+Highcharts.chart('container', {
+
+    plotOptions: {
+        series: {
+            animation: {
+                duration: 2000
+            }
+        }
+    },
+
+    series: [{
+        data: [{
+            y: 29.9,
+            id: 'min'
+        }, 71.5, 106.4, 129.2, 144.0, 176.0, 135.6, 148.5, {
+            y: 216.4,
+            id: 'max'
+        }, 194.1, 95.6, 54.4]
+    }],
+
+    annotations: [{
```

Comment:
Could you split this annotation into three separate ones (label with max, label with min, just a shape) - and set different options (`defer=true`, `defer=false`, `defer={duration}`)?

Setting title for this demo would be nice too :)